### ISSUE TITLE
Search Adjustments

### ISSUE THREAD

**Issue by 0x4007** [ubiquity/work.ubq.fi#179]
I was searching for https://devpool.directory/?search=top+up "top up" and its actually in the title of the task I was searching for. We need to increase the weight of the following:

1. task title
2. repository name
3. organization name

I think it could be interesting to prefix special operators to hone in for example imagine typing `@ubiquity/business-development` and then the UI would understand that you want to search only in the scope of that repository. This can be a shorthand hack instead of changing the url to `devpool.directory/ubiquity` but it would also make it clear that we are searching within a repository. Otherwise I suppose we can see if the search begins with a `/` and then it can signal that you are targeting the repository. 

In this case we can search `/business-development` and it would isolate the tasks from that repository. 

In conclusion we should 1. increase the weight of matching task titles and 2. we should allow these fine grained search filters by parsing the symbols `@` and `/` in the first character of the search.

**Comment by zugdev** [comment]
@0x4007 @gentlementlegen should we use pricing from development branch? I think main is down?

**Comment by 0x4007** [comment]
This repo uses a different config so it must be wrong compared to our org configs.

**Comment by zugdev** [comment]
True, it looks like it's using old assistive pricing

**Comment by 0x4007** [comment]
Feel free to open a pull to fix

**Comment by zugdev** [comment]
@0x4007 done in #182

**Comment by ishowvel** [comment]
@0x4007 for the second and thir we can just 

> I was searching for https://devpool.directory/?search=top+up "top up" and its actually in the title of the task I was searching for. We need to increase the weight of the following:
>     2. repository name
> 
>     3. organization name

for the second and third we can just use the / and @ prefixes respectively as you mentioned

>    1. task title

searching using a keyword of any task titles currently yields that respective issue at the top currently so i dont see how thats an issue

**Comment by 0x4007** [comment]
@gentlementlegen any ideas? Should be do a backup not relying on deno somehow?

**Comment by gentlementlegen** [comment]
We could eventually have a backup but I am not sure of the root cause of the problem. I have no trouble running it locally. I will try to debug this today, and let you know if I find something.
